docs/en/setup/service-agent/java-agent/configurations.md

```diff
@@ -60,7 +60,7 @@ This is the properties list supported in `agent/config/agent.config`.
 | `plugin.peer_max_length `                                       | Peer maximum description limit.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        | SW_PLUGIN_PEER_MAX_LENGTH                                        | `200`                                                                   |
-| `plugin.exclude_plugins `                                       | Exclude some plugins define in plugins dir,Multiple plugins are separated by comma.Plugin names is defined in [Agent plugin list](Plugin-list.md)                                                                                                                                                                                                                                                                                                                                                                                                                                              | SW_EXCLUDE_PLUGINS                                               | `""`                                                                    |
+| `plugin.exclude_plugins `                                       | Exclude some plugins define in plugins dir,Multiple plugins are separated by comma.Plugin names is defined in [Agent plugin list](Plugin-list.md)                                                                                                                                                                                                                                                                                                                                                                                                      | SW_EXCLUDE_PLUGINS                                               | `""`                                                                    |
```

Comment:
What is changed in this line?

Reply:
Noting, I will reset and rewrite the file .